spotbugs/etc/messages.xml

```diff
@@ -8280,18 +8273,18 @@ and <a href="http://bugs.java.com/bugdatabase/view_bug.do?bug_id=6178997">JDK Bu
     <Details>
       <![CDATA[
       <p>
-     This code opens a file in append mode and then wraps the result in an object output stream.
+     This code opens a file in append mode and then wraps the result in an object output stream:
+      </p>
+      <pre><code>OutputStream out = new FileOutputStream(anyFile, true);
+     new ObjectOutputStream(out);</code></pre>
+      <p>
      This won't allow you to append to an existing object output stream stored in a file. If you want to be
      able to append to an object output stream, you need to keep the object output stream open.
       </p>
       <p>The only situation in which opening a file in append mode and the writing an object output stream
       could work is if on reading the file you plan to open it in random access mode and seek to the byte offset
       where the append started.
       </p>
-
-      <p>
-      TODO: example.
-      </p>
```

Comment:
Better to replace with an example, instead of a simple deletion.

Reply:
I added the example some lines above (line 8276)